stable/sonarqube/values.yaml

```diff
@@ -138,6 +138,7 @@ persistence:
 #    - "https://github.com/SonarSource/sonar-ldap/releases/download/2.2-RC3/sonar-ldap-plugin-2.2.0.601.jar"
 plugins:
   install: []
+  list: []
```

Comment:
This doesn't match the file above. Please change to 
`plugins:
    lib: []
`

Reply:
Thanks, @rjkernick!